src/lang/bg-BG.json

```diff
@@ -326,6 +326,7 @@
   "heading_graph_gold": "Злато",
   "heading_graph_xp": "Опит",
   "heading_graph_lh": "Последни удари",
+  "heading_graph_cs": "Последни удари + Denies",
```

Comment:
You can revert changes to all lang files besides English, strings will be added automatically later.